### ISSUE TITLE
Please remove postinstall message about donating

### ISSUE THREAD

**Issue by eddiemonge** [cssinjs/jss#881]
While I understand the need to ask for donations, having it as the [postinstall message](https://github.com/cssinjs/jss/blob/master/packages/jss/package.json#L42), to me, isn't the most ideal location.

Downsides of having it there:
- It clutters up server logs (it does not respect the `--quiet` flag
- It is excessive (4 lines long)
- It sets a bad precedent/example. If every package did this, there would be hundreds of these messages.

This type of message is better suited to be at the top of the repo (hint hint), in the readme file and on the website.

**Comment by kof** [comment]
Actually I don't think it has yeld any positive results, so I think this experiment should end.  I got the idea from preact, they are still doing it though https://github.com/developit/preact/blob/master/package.json#L40

**Comment by eddiemonge** [comment]
I saw that in the commit where it got added here. Or a google search. I hope it's not openCollective's idea. If so, please let me know so I can have a word with them (nice people)

**Comment by kof** [comment]
I don't know whos initial idea it is, I discovered it on preact, and later on I saw it in some other quite popular projects.

**Comment by kof** [comment]
if you search on github for `"postinstall": "node -e \"console.log('\\u001b[35m\\u001b[1m` it seems like quite many projects do that by now

**Comment by eddiemonge** [comment]
There are a couple of packages that do it. I wonder if any of them obey npm's loglevel. If not, maybe one should be made:
```js
const loglevels = ["silent", "error", "warn"]
if (logLevels.indexOf(process.env.npm_config_loglevel) < 0) { showPostInstall(); }
```

**Comment by kof** [comment]
same question, or alternatively create a script that can log that stuff and respect the log level

**Comment by eddiemonge** [comment]
Perhaps this could switch to use https://github.com/opencollective/opencollective-postinstall ?

**Comment by kof** [comment]
Oh it already exists, does it respect log level? I suppose not.

**Comment by eddiemonge** [comment]
It doesn't but I might PR for it. They use a specific flag to hide the message, which unless you know about the flag, isn't very helpful

**Comment by kof** [comment]
Well it is helpful, since you would do it only once for all projects using it, yes you have to discover it and it should respect log level, I think they will be happy to merge a PR and JSS can switch to that script.

**Comment by kof** [comment]
Will be released with v10, follow #795
